File: github.py
from selenium import webdriver


if __name__ == '__main__':

    # Chrome WebDriver 로 chrome.exe 실행
    browser = webdriver.Chrome('./chromedriver.exe')

    # 사이트 호출
    browser.implicitly_wait(10)
    browser.get("https://github.com/ImDaeseong")

    # 페이지 읽기
    print(browser.page_source)

    # 드라이브 종료
    browser.quit()

File: golang/github.go
// github
package main

import (
	"fmt"
	"time"

	"github.com/tebeka/selenium"
)

const (
	path = `c:\GoApp\chromedriver.exe`
	port = 8080
)

func main() {

	opts := []selenium.ServiceOption{}

	server, err := selenium.NewChromeDriverService(path, port, opts...)
	if err != nil {
		fmt.Println(err)
	}
	defer server.Stop()

	c := selenium.Capabilities{"browserName": "chrome"}

	wd, err := selenium.NewRemote(c, fmt.Sprintf("http://localhost:%d/wd/hub", port))
	if err != nil {
		fmt.Println(err)
	}
	defer wd.Quit()

	err = wd.Get("https://github.com/ImDaeseong")
	if err != nil {
		panic(err)
	}

	url, _ := wd.CurrentURL()
	title, _ := wd.Title()

	fmt.Println(url)
	fmt.Println(title)

	time.Sleep(10 * time.Second)
}
